<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>791</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
       <item>
        <layout class="QVBoxLayout" name="sceneOptions">
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="loadModelBtn">
         <property name="text">
          <string>Load Model</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="GLWidget" name="openglWidget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuScene">
    <property name="title">
     <string>Scene</string>
    </property>
    <addaction name="actionScenePhong"/>
   </widget>
   <addaction name="menuScene"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionScenePhong">
   <property name="text">
    <string>Phong</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header location="global">glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
